recipes/daggy/2.1/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.21.0)
+project(DaggyPackageTest)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(daggy REQUIRED)
+
+if (NOT CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 17)
+endif()
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_subdirectory(testcpp)
```

Comment:
```suggestion
```

Keep all under test_package folder, don't need to create subfolders for a simple test.

Reply:
Moved test from subdirectory